webapp/src/app.tsx

```diff
@@ -2793,7 +2793,12 @@ export class ProjectView
         return this.createProjectAsync(options)
             .then(() => this.autoChooseBoardAsync())
             .then(() => Util.delay(500))
-            .finally(() => core.hideLoading("newproject"));
+            .finally(() => {
+                core.hideLoading("newproject");
+                if (options?.firstProject) {
```

Comment:
Ah wait, one more thing; this should probably go behind a target flag, in case a target doesn't have editor tours (or has an editor tour but doesn't want it automatic)

Reply:
Oh yea good catch!